internal/config/config.go

```diff
@@ -42,6 +42,8 @@ type Config struct {
 	LogLevel              string                  `default:"INFO" desc:"Log level" split_words:"true"`
 	OpenTelemetryEndpoint string                  `default:"otel-collector.observability.svc.cluster.local:4317" desc:"OpenTelemetry Collector Endpoint"`
 
+	DefaultCorefilePath string `default:"/etc/coredns/Corefile" desc:"Default path to the Corefile source"`
```

Comment:
If this is the CoreDNS config file could we name it something that indicates that?  "CorefilePath" sounds like where we would write coredumps :)

Reply:
Agreed, fixed